sdk/azcore/runtime/transport_default_http_client.go

```diff
@@ -22,8 +22,13 @@ func init() {
 			Timeout:   30 * time.Second,
 			KeepAlive: 30 * time.Second,
 		}).DialContext,
-		ForceAttemptHTTP2:     true,
-		MaxIdleConns:          100,
+		ForceAttemptHTTP2: true,
+		MaxIdleConns:      100,
+		//https://pkg.go.dev/net/http#Transport
+		// MaxIdleConnsPerHost, if non-zero, controls the maximum idle
+		// (keep-alive) connections to keep per-host. If zero,
+		// DefaultMaxIdleConnsPerHost(2) is used.
+		MaxIdleConnsPerHost:   100,
```

Comment:
Do you think this is a reasonable default for _all_ clients (i.e. there are no negative side effects)?

You can create your own client transport and specify it [here](https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/azcore/policy/policy.go#L46) when constructing clients.

Reply:
I think it's reasonable because we found this config will have huge impact on blob client when there are a lot of connections.